test/infrastructure/docker/config/default/namespace.yaml

```diff
@@ -3,4 +3,7 @@ kind: Namespace
 metadata:
   labels:
     control-plane: controller-manager
```

Comment:
Let's add a short comment why we need privileged

Reply:
Added :-)